### DISCUSSION TITLE
RefreshView

### DISCUSSION THREAD

**Discussion by korpus2** [dotnet/maui#26384] [Q&A]
Hi,
is there any way in maui to make custom arrow spinner or delete that? Just to fire refresh event when we pull down? I'm using CollectionView and I would like to use own behavior of refreshing. 
Thx ;)

**Comment by Yannikk1996** [comment]
I would also like to know if it is possible to replace the default indicator in the RefreshView component with a custom element, such as an SVG or a Skia/Lottie animation.
